pkg/tag/accounting.go

```diff
@@ -0,0 +1,10 @@
+package tag
+
+const (
+	// AccountingID tag to store an accounting ID
+	AccountingID = "accounting.metal-stack.io/id"
```

Comment:
After re-checking the accounting-client I wonder what the "accounting ID" is. I found, that in you constants "accounting.metal-stack.io/productid" is missing, maybe you wanted to address this by "id"?

Reply:
I guess I was just not very precise, it's better to use productid, I think. I made a follow-up in #24 because this was already merged.